### ISSUE TITLE
Settings screen redux

### ISSUE THREAD

**Issue by AhoyLemon** [AnotherDole/wrongest#64]
Going to look into if it's a good idea to bring some sort of focus to the settings screen just so it's not overlooked.

**Comment by AhoyLemon** [comment]
Wanna try two a couple more things as part of this. 

1. Add round limit to the options.
2. Try to make "redraw same card?" always fit on a single line. 
3. Try to prevent the gear from flying up when the OSD keyboard is up (again)

**Comment by AhoyLemon** [comment]
Also,  kind of unrelated,  but cut down the opening screen padding so it won't scroll as much on a phone.

**Comment by AhoyLemon** [comment]
Okay, I have these changes in a  branch called `setup-redux`, here's what I changed.

1. Made the settings gear blue.
2. Added a select for "round limit"... "NEVER" is probably a bad idea in hindsight.
3. Commented out "redraw same card?" entirely? I feel like it's an option without a whole lot of value and I do think it creates confusion as far as what it means.
4. Added a new height breakpoint that will change the admin-settings to `position:relative` when the height is less than 320 pixels, which looks a little weird, but it should really only happen if you've got your OSD keyboard up.

I put this all in a branch because I don't have any logic in there for "round limit" to actually change anything.
